pom.xml

```diff
@@ -108,17 +108,29 @@
 
     <!-- ONS libraries -->
 
+    <dependency>
```

Comment:
We could get away without this framework dependency since it is a dependency elsewhere.

To avoid having to add the case-api-client and the event-publisher we could have updated the eq-launcher, but I guess that is for another time.

Reply:
Our changes crossed and the pom dependencies are as they used to be (except for bumping versions).
The explicit, and arguably unnecessary, dependencies have the advantage of speeding up the review process as there is of course one less round of reviewing to be done.
I would have thought that this is fine for development but it should all be cleaned up, with everything on the latest of everything else, before going into serious testing.